app/views/courses/TeacherClassView.coffee

```diff
@@ -171,7 +177,8 @@ module.exports = class TeacherClassView extends RootView
     null
 
   onLoaded: ->
-    @sortedCourses = @classroom.getSortedCourses()
+    @latestCourses = @courses.where({releasePhase: 'released'})
+    utils.sortCourses(@latestCourses)
```

Comment:
Will rename `@latestReleasedCourses` and add a comment.  I created a new variable because I only wanted the released courses, but did not want to figure out all the `@courses` uses.